pipelines/coref/project.yml

```diff
@@ -0,0 +1,133 @@
+title: "Training a spaCy Coref Model"
+description: >
+  This project trains a coreference model for spaCy using OntoNotes.
+
+spacy_version: ">=3.3.0,<4.0.0"
+
+vars:
+  # XXX Change to your actual GPU ID
+  gpu_id: 0
+  # XXX fill with your local path to OntoNotes
+  ontonotes: /home/USER/ontonotes5/data
+  max_epochs: 20
+
+directories: ["assets", "corpus", "scripts", "configs", "training"]
+
+assets:
+  - dest: assets/
+    git:
+      repo: "https://github.com/explosion/conll-2012"
+      branch: "main"
+      path: ""
+    description: "CoNLL-2012 scripts and dehydrated data."
+  - dest: ${vars.ontonotes}
+    description: "Ensure you have a local copy of OntoNotes: https://catalog.ldc.upenn.edu/LDC2013T19"    
+
+workflows:
+  prep:
+    - prep-data
+    - preprocess
+  train:
+    - train-cluster
+    - prep-span-data
+    - train-span-predictor
+    - assemble
+
+  all:
+    # prep
+    - prep-data
+    - preprocess
+    # train
+    - train-cluster
+    - prep-span-data
+    - train-span-predictor
+    - assemble
+    # eval
+    - eval
+
+commands:
+  - name: "prep-data"
+    help: "Rehydrate the data using OntoNotes"
+    script: 
+      - tar xzf assets/conll-2012-development.v4.tar.gz -C assets/
+      - tar xzf assets/conll-2012-test-key.tar.gz -C assets/
+      - tar xzf assets/conll-2012-test-official.v9.tar.gz -C assets/
+      - tar xzf assets/conll-2012-train.v4.tar.gz -C assets/
+
+      - echo "Rehydrating data... This will take roughly 20 minutes"
+      - "bash assets/conll-2012/v3/scripts/skeleton2conll.sh -D ${vars.ontonotes} assets/conll-2012"
+
+      # Concatenate the data into single files to make it easier to work with
+      - sh -c "rm -f assets/*._gold_conll"
+      - sh -c "cat assets/conll-2012/v4/data/development/data/english/annotations/*/*/*/*.v4_gold_conll >> assets/dev.gold.conll"
+      - sh -c "cat assets/conll-2012/v4/data/train/data/english/annotations/*/*/*/*.v4_gold_conll >> assets/train.gold.conll"
+      - sh -c "cat assets/conll-2012/v4/data/test/data/english/annotations/*/*/*/*.v4_gold_conll >> assets/test.gold.conll"
+    deps:
+      - ${vars.ontonotes}
+    outputs: 
+      - assets/train.gold.conll
+      - assets/dev.gold.conll
+      - assets/test.gold.conll
+
+  - name: "preprocess"
+    help: "Convert the data to spaCy's format"
+    deps:
+      - assets/train.gold.conll
+      - assets/dev.gold.conll
+      - assets/test.gold.conll
+    script:
```

Comment:
```suggestion
    script:
        - python -m spacy download en_core_web_lg
```

Reply:
Currently we are specifying models in `requirements.txt`. Still not clear if that's the best way though.

Reply:
Marking resolved because this is using the large one.